### PR TITLE
UI: resolve .url, .lnk shortcuts during drag-and-drop

### PR DESCRIPTION
**Pull request by bontibon**
### Description

Resolve .url, .lnk files on Windows when dragged onto the main window.

### Motivation and Context

Suggested here: https://ideas.obsproject.com/posts/1636

### How Has This Been Tested?

Tested on Windows 10 by dragging different types of files onto the main Window.

### Types of changes

Tweak

### Checklist:

- [X] My code has been run through [clang-format](https://github.com/obsproject/obs-studio/blob/master/.clang-format).
- [X] I have read the [**contributing** document](https://github.com/obsproject/obs-studio/blob/master/CONTRIBUTING.rst).
- [X] My code is not on the master branch.
- [X] The code has been tested.
- [X] All commit messages are properly formatted and commits squashed where appropriate.
- [X] I have included updates to all appropriate documentation.

### CODE REVIEW

File: UI/window-basic-main-dropfiles.cpp [hunk 1]
```diff
@@ -5,6 +5,7 @@
 #include <QFileInfo>
 #include <QMimeData>
 #include <QUrlQuery>
+#include <QSettings>
```
Comment (WizardCM) [inline UI/window-basic-main-dropfiles.cpp:8]: If the rest of the code is behind a `#ifdef _WIN32` then this should be too.

Comment (bontibon) [reply UI/window-basic-main-dropfiles.cpp:9]: Fixed.

**Comment by bontibon** [comment]
Rebased.

**Comment by therentabrain** [comment]
Noob question:  I just updated and I guess there's some review required still, because the shortcut drop still doesn't work.  Is there anything I can do to help it happen?  Thank you so much.  I'm sure things are hectic with the push rollout and such.  I love OBS with my whole stream.

**Comment by RytoEX** [comment]
> Noob question: I just updated and I guess there's some review required still, because the shortcut drop still doesn't work. Is there anything I can do to help it happen? Thank you so much. I'm sure things are hectic with the push rollout and such. I love OBS with my whole stream.

This has not been merged, so it is not present in any currently distributed version of OBS.  Enhancements or new features typically are only merged for minor or major releases, and even then only if we finish our reviews and testing of it and find it acceptable to merge.  Thank you for your patience.

**Pull request merged**